infrastructure/src/main/java/org/corfudb/infrastructure/ServerContext.java

```diff
@@ -189,6 +192,13 @@ public ServerContext(Map<String, Object> serverConfig) {
         if (!isMetricsReportingSetUp(metrics)) {
             MetricsUtils.metricsReportingSetup(metrics);
         }
+
+        FileChannelPerf.setupParameters(OP_SPIKE_CNT, OP_SPIKE_LATENCY, false);
+        logMetricsSize();
+    }
+
+    public void logMetricsSize() {
+        log.debug("corfu server metrics size {}.", MetricsUtils.sizeOf.deepSizeOf(ServerContext.getMetrics()));
```

Comment:
What's the point of this?

Reply:
I just wanted to see the size of the metrics.